collaborations/blood-hunter-2022/bloodhunter-profanesoul.xml

```diff
@@ -0,0 +1,654 @@
+			<p class="indent"><strong><em>The Undead.</em></strong> When you take necrotic damage, you can use your reaction to halve that damage. In addition, your appearance changes to reflect some aspect of your patron while you have any crimson rite active.</p>
+			<p class="indent"><strong><em>The Undying.</em></strong> When you reduce a hostile creature of at least mild threat (DM’s discretion) to 0 hit points, you regain a number of hit points equal to one roll of your hemocraft die.</p>
+		</description>
+		<sheet>
+			<description>While you have an active Crimson Rite, you can use your weapon as a spellcasting focus for your warlock spells.</description>
+		</sheet>
+		<rules>
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_ARCHFEY" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_ARCHFEY" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_CELESTIAL" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_CELESTIAL" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_FATHOMLESS" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_FATHOMLESS" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_FIEND" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_FIEND" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_GENIE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_GENIE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_GREAT_OLD_ONE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_GREAT_OLD_ONE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_HEXBLADE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_HEXBLADE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_UNDEAD" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_UNDEAD" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_RITE_FOCUS_UNDYING" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_UNDYING" />
+		</rules>
+	</element>
+	
+	<element name="Mystic Frenzy" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_MYSTIC_FRENZY">
+		<description>
+			<p>Starting at 7th level, when you use your action to cast a cantrip, you can immediately make one weapon attack as a bonus action.</p>
+		</description>
+		<sheet action="Bonus Action">
+			<description>When you use your action to cast a cantrip, you can immediately make one weapon attack.</description>
```

Comment:
need to add "as a bonus action." to the ending otherwise, it is assumed they get one free weapon attack.